### ISSUE TITLE
ADSR2 Help File needs more...

### ISSUE THREAD

**Issue by darwingrosse** [stretta/BEAP#74]
The help file for ADSR2 needs additional explanation of the live.numboxes on the interface - no explanation (of the curve, I guess?) is provided.

**Comment by stretta** [comment]
Yes, curve. Added some text. The module does provide hover-over description of these values.

**Comment by darwingrosse** [comment]
Hover-over how? Oh, only if you have a clue window available...

**Comment by stretta** [comment]
ah, sorry, I was thinking of VC-ADSR, not ADSR2
